code/modules/clothing/under/color.dm

```diff
@@ -182,24 +182,28 @@
 	_color = "blpants"
 	clothing_flags = ONESIZEFITSALL
 	gender = PLURAL
+	body_parts_covered = LOWER_TORSO|LEGS
 
 /obj/item/clothing/under/redpants
 	name = "red pants"
 	icon_state = "rpants"
 	_color = "rpants"
 	clothing_flags = ONESIZEFITSALL
 	gender = PLURAL
+	body_parts_covered = LOWER_TORSO|LEGS
 
 /obj/item/clothing/under/bluepants
 	name = "blue pants"
 	icon_state = "bpants"
 	_color = "bpants"
 	clothing_flags = ONESIZEFITSALL
 	gender = PLURAL
+	body_parts_covered = LOWER_TORSO|LEGS
```

Comment:
It makes it so that wearing only pants doesn't give you the same level of protection from cockatrices as a jumpsuit

Although it may affect other stuff (I believe chemicals), so I'll add it to CL